x/savings/keeper/deposit.go

```diff
@@ -0,0 +1,60 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/kava-labs/kava/x/savings/types"
+)
+
+// Deposit deposit
+func (k Keeper) Deposit(ctx sdk.Context, depositor sdk.AccAddress, coins sdk.Coins) error {
+
+	err := k.ValidateDeposit(ctx, coins)
+	if err != nil {
+		return err
+	}
+
+	err = k.bankKeeper.SendCoinsFromAccountToModule(ctx, depositor, types.ModuleAccountName, coins)
+	if err != nil {
+		return err
+	}
+
+	currDeposit, foundDeposit := k.GetDeposit(ctx, depositor)
+	var amount sdk.Coins
+	if foundDeposit {
+		amount = currDeposit.Amount.Add(coins...)
+	} else {
```

Comment:
can ignore this else if you initialize as `amount := coins`

Reply:
Updated in https://github.com/Kava-Labs/kava/pull/1192/commits/153393d4a9ba56d21627f8f597a2da860a2faf3a